scripts/e2e-test.sh

```diff
@@ -78,3 +78,28 @@ helm template deploy/helm/eunomia-operator/ \
   --set eunomia.operator.image.tag=dev \
   --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl delete -f -
 
+# Below block is to ensure hello-world-yaml example is working
+helm template deploy/helm/eunomia-operator/ \
+  --set eunomia.operator.image.tag=dev \
+  --set eunomia.operator.image.pullPolicy=Never | kubectl apply -f -
+
+kubectl create namespace eunomia-hello-world-yaml-demo
```

Comment:
It might be a good idea to adding an `echo` before each command, in case it fails. Otherwise it'll be harder to debug.
E.g. `echo creating namespace xxx`

Reply:
The script has `set -x` at the beginning, so I think it should be enough?

Reply:
Good point.